scripts/zones/Riverne-Site_B01/npcs/Unstable_Displacement.lua

```diff
@@ -11,21 +13,15 @@ require("scripts/globals/bcnm")
 local entity = {}
 
 entity.onTrade = function(player, npc, trade)
-    local offset = npc:getID() - ID.npc.DISPLACEMENT_OFFSET
-    if (offset == 5 and TradeBCNM(player, npc, trade)) then -- The Wyrmking Descends
-        return
-    end
+    TradeBCNM(player, npc, trade)
```

Comment:
sooo we don't need to know _which_ unstable displacement it is before hitting the BCNM code anymore?

Reply:
from discord:
> Re: the unstable displacement change in the PR, looks like there's only one "officially named" unstable displacement in B01 now:
INSERT INTO npc_list VALUES (16896217,'Unstable_Displacement','Unstable Displacement',0,-612.800,1.750,693.190,32769,50,50,0,0,0,0,3,0x0000320000000000000000000000000000000000,0,'COP',0);

Reply:
Yup!  This NPC is the only one with name = 'Unstable_Displacement' in the zone.  The other three "Unstable Displacement" NPCs in the zone are `_0t1`, `_0t2`, and `_0t3`.

Reply:
Not sure how long ago it was they changed, but waaay back when they were brand new it was all one giant script for them all lol. In _both_ rivernes (1 script for each zone).